libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/loadDashboardHandler/loadCatalog.ts

```diff
@@ -1,9 +1,26 @@
 // (C) 2021 GoodData Corporation
 import { DashboardContext } from "../../../types/commonTypes";
-import { IWorkspaceCatalog } from "@gooddata/sdk-backend-spi";
+import { IWorkspaceCatalog, IWorkspaceCatalogFactoryOptions } from "@gooddata/sdk-backend-spi";
 
 export function loadCatalog(ctx: DashboardContext): Promise<IWorkspaceCatalog> {
     const { backend, workspace } = ctx;
 
-    return backend.workspace(workspace).catalog().load();
+    const options: IWorkspaceCatalogFactoryOptions = {
+        excludeTags: [],
+        includeTags: [],
+        types: ["attribute", "fact", "measure", "dateDataset"],
+        includeDateGranularities: [
+            "GDC.time.day_in_week",
+            "GDC.time.day_in_month",
+            "GDC.time.day_in_quarter",
+            "GDC.time.day_in_year",
+            "GDC.time.week_in_quarter",
+            "GDC.time.week_in_year",
+            "GDC.time.month_in_quarter",
+            "GDC.time.month_in_year",
+            "GDC.time.quarter_in_year",
+        ],
```

Comment:
please move this into a constant typed as `DateAttributeGranularity[]`...

the catalog includeDateGranularities option is currently not typed well as it accepts string[] but should be stricter :( however we can keep the client code safe by having correctly typed constant

Reply:
Done